test/parallel/test-module-readonly.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs');
+const path = require('path');
+const cp = require('child_process');
+
+const fixtures = require('../common/fixtures');
+
+if (process.platform == 'win32') {
+    // Create readOnlyMod.js and set to read only
+    const readOnlyMod = fixtures.path('readOnlyMod');
```

Comment:
I think the sum total of the change would be:

* require `tmpdir` instead of `fixtures`
* call `tmpdir.refresh()` once
* Use `tmpdir.path()` instead of `fixtures.path()`